deps/pcre.mk

```diff
@@ -6,6 +6,13 @@ ifneq ($(USE_BINARYBUILDER_PCRE),1)
 PCRE_CFLAGS := -O3
 PCRE_LDFLAGS := $(RPATH_ESCAPED_ORIGIN)
 
+ifeq ($(OS),emscripten)
+PCRE_CFLAGS += -fPIC
+PCRE_JIT = --disable-jit
```

Comment:
I was under the impression that the PCRE JIT is not optional for us.  Is that no longer true?

Reply:
It is optional, just a bit slower without it